src/index.js

```diff
@@ -177,38 +144,51 @@ const mountVue = (component, optionsOrProps = {}) => () => {
     props = optionsOrProps
   }
 
-  const vueHtml = getPageHTML(options)
-  const document = cy.state('document')
-  document.write(vueHtml)
-  document.close()
-
-  // TODO: do not log out "its(Vue)" command
-  // but it currently does not support it
-  return cy
-    .window({ log: false })
-    .its('Vue')
-    .then(Vue => {
-      // refresh inner Vue instance of Vuex store
-      if (hasStore(component)) {
-        component.store = resetStoreVM(Vue, component)
-      }
-      installMixins(Vue, options)
-      installPlugins(Vue, options)
-      registerGlobalComponents(Vue, options)
-      deleteCachedConstructors(component)
-
-      if (isConstructor(component)) {
-        const Cmp = Vue.extend(component)
-        Cypress.vue = new Cmp(props).$mount('#app')
-        copyStyles(Cmp)
-      } else {
-        const allOptions = Object.assign({}, component, { el: '#app' })
-        Cypress.vue = new Vue(allOptions)
-        copyStyles(component)
-      }
-
-      return Cypress.vue
-    })
+  // display deprecation warnings
+  if (options.vue) {
+    console.warn(stripIndent`
+      [DEPRECATION]: 'vue' option has been deprecated.
+      'node_modules/vue/dis/vue' is always used.
+      Please remove it from your 'mountVue' options.`)
+  }
+
```

Comment:
nice touch - showing deprecation warning

Reply:
I'm all about smooth transitions from version to version.